appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/BaseTransitionModel.kt

```diff
@@ -53,6 +55,17 @@ abstract class BaseTransitionModel<NavTarget, ModelState>(
         removeDestroyedElements()
     }
 
+    override fun onAnimationFinished(navElement: NavElement<NavTarget>) {
+        state.getAndUpdate { output ->
+            when (output) {
+                is Update<ModelState> -> output.copy(
+                    currentTargetState = output.currentTargetState.removeDestroyedElement(navElement)
+                )
+                is Keyframes -> output
```

Comment:
Can this ever be a valid case where we'd do something similar here too?

Reply:
yeah but we already have `fun onAnimationFinished()`